radio/src/gui/480x272/theme.h

```diff
@@ -26,7 +26,7 @@ class BitmapBuffer;
 #define MAX_THEME_OPTIONS              5
 
 class Theme;
-void registerTheme(Theme * theme);
+extern void registerTheme(Theme * theme);
```

Comment:
Yes I remember in er9x all functions were marked as extern until someone raised an issue, I started to remove them little after little at this time